### PR TITLE
aligned manpages with scale-up

### PR DESCRIPTION
**Pull request by lpinne**
aligned manpages with scale-up
based on feedback from partners and customers

### CODE REVIEW

File: SAPHana/man/ocf_suse_SAPHanaTopology.7 [hunk 1]
```diff
@@ -1,12 +1,12 @@
-.\" Version: 0.181.0
+.\" Version: 0.160.1
```
Comment (fmherschel) [inline SAPHana/man/ocf_suse_SAPHanaTopology.7:1]: @lpinne : Is this downgrade really intended?

File: SAPHana/man/SAPHanaSR_maintenance_examples.7 [hunk 2]
```diff
@@ -1,6 +1,6 @@
-.\" Version: 0.181.0
+.\" Version: 0.160.1
```
Comment (fmherschel) [inline SAPHana/man/SAPHanaSR_maintenance_examples.7:1]: @lpinne : Is this downgrade really intended?

**Review by fmherschel — COMMENTED** [review]
@lpinne : I have two questions regarding the downgrade of the man-page version number.
Also I like to ask, is it intended that the PR is against master/main ?
@angelabriel As this "only" changes man pages, is it ok to directly go to master/main, or do you like/need to sample such commits?

**Comment by angelabriel** [comment]
> @angelabriel As this "only" changes man pages, is it ok to directly go to master/main, or do you like/need to sample such commits?

For the alignment of the man pages I think it is ok to work directly on the master branch, because we will add these changes to the next maintenance update in any case.

**Pull request merged**